### PR TITLE
OODT-1003 Upgrade Apache parent POM to version 21

### PR DESCRIPTION
**Pull request by lewismc**
This issue addresses https://issues.apache.org/jira/browse/OODT-1003

### CODE REVIEW

**Comment by lewismc** [comment]
By the way, this issue also fixes a NoDefClassFound Exception which arises in Workflow Manager due to a method return value change in Lucene's IndexWriter.addDocuement method. This occurs when you use Lucene as the data store for capturing workflow information.

**Comment by lewismc** [comment]
Any objections to merge?

**Comment by chrismattmann** [comment]
@lewismc master/1.9 is the main dev branch. I keep development branch in sync as well usually as a push from master.

File: workflow/src/test/java/org/apache/oodt/cas/workflow/system/TestAvroRpcWorkflowManager.java [hunk 1]
```diff
@@ -31,7 +37,7 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-public class TestAvroRpcWorkflowManager extends TestCase {
+public class TestAvroRpcWorkflowManager {
```
Comment (chrismattmann) [inline workflow/src/test/java/org/apache/oodt/cas/workflow/system/TestAvroRpcWorkflowManager.java]: this change seems orthogonal to this PR, can you remove?

**Comment by IMS94** [comment]
@lewismc If you can resolve @chrismattmann's comment, I think we can merge this. Could you look into that?

**Comment by buggtb** [comment]
I'm gonna merge this and fix it post merge to get it closed.
